karaf-features/src/main/resources/features.xml

```diff
@@ -286,4 +287,14 @@
         <bundle>mvn:org.codehaus.groovy/groovy-all/${groovy.version}</bundle>
     </feature>
 
+    <feature name="resilience4j" version="${resilience4j.version}" description="resilience4j">
```

Comment:
Also add `resilience4j` to this list here: https://github.com/OpenNMS/alec/blob/3b592c31df21ae6cbf368e8e84ffd6d08014e316/karaf-features/pom.xml#L125

Reply:
:+1: done